### PR TITLE
events.html: Fix thumb for upcoming events; complement river

### PR DESCRIPTION
**Pull request by earthboundkid**

### CODE REVIEW

File: layouts/_default/events.html [hunk 1]
```diff
@@ -9,13 +9,12 @@
   {{ $p := partial "helper/paginator" . }}
   {{ $pages := $p.Pages }}
   {{ $upcomingEvents := where $pages `Params.event-date` "gt" now }}
-  {{ $upcomingEvents = sort $upcomingEvents `Params.event-date` "desc" }}
+  {{ $upcomingEvents = sort $upcomingEvents `Params.event-date` "asc" }}
```
Comment (earthboundkid) [inline layouts/_default/events.html:12]: I think we want the soonest event at the top of the page.

File: layouts/_default/events.html [hunk 2]
```diff
@@ -51,68 +68,90 @@ <h2 class="text-2xl font-extrabold">
...
-    {{ range after 1 . }}
-      {{ $params := partialCached "helper/page-params" . .RelPermalink }}
-      {{ $subitems = $subitems | append (dict
-        "link" $params.link
-        "title" .Title
-        )
-      }}
-    {{ end }}
-    {{ $imgOpts := dict
-      "filename" $params.image
```
Comment (earthboundkid) [inline layouts/_default/events.html:83]: This was a bug because the $params object doesn't have image on it. It's on the page.Params.

Comment (jeffrummel) [reply layouts/_default/events.html:83]: I'm not sure I understand the different use-cases for `$params` vs just `$event`

**Review by jeffrummel — APPROVED** [review]

File: layouts/_default/events.html (hunk 2 above)
Comment (earthboundkid) [reply layouts/_default/events.html:83]: $params is the `partialCached "helper/page-params"` object that does some logic for us of calculating the pretty event date, author name, eyebrow links, etc. It exists so we have one place that does the fallback logic for that stuff when a parameter is missing or needs to fallback to different stuff. $event.Params is the native Hugo params dictionary.

**Pull request merged**